packages/nx-playwright/src/generators/init/lib/add-git-ignore-entry.ts

```diff
@@ -0,0 +1,27 @@
+import { Tree } from '@nrwl/devkit';
```

Comment:
```suggestion
import { Tree } from '@nrwl/devkit';
import ignore from 'ignore';

export const addGitIgnoreEntry = (host: Tree) => {
  const content = host.exists('.gitignore') ? host.read('.gitignore', 'utf-8').trimEnd() : '';

  const ig = ignore();
  ig.add(host.read('.gitignore', 'utf-8'));

  const addIgnore = (existingIgnore: string, ignoreToAdd: string) => {
    return ig.ignores(existingIgnore) ? '' : ignoreToAdd;
  };

  const resultsIgnore = addIgnore('app/example/test-results', `\n\n# Playwright\n**/test-results/`);
  const reportsIgnore = addIgnore('app/example/playwright-report', `\n**/playwright-report/`);
  const cacheIgnore = addIgnore('app/example/playwright/.cache', '\n**/playwright/.cache/\n');

  host.write('.gitignore', `${content}${resultsIgnore}${reportsIgnore}${cacheIgnore}`);
};

```

Reply:
thanks, very nice cleanup @andreacaldera!

Reply:
since your suggested change added code rather than replaced it, when I removed the extra function I accidentally removed yours @andreacaldera .  re-added & thanks again